bundles/mapping/mapwfs2/plugin/WfsMvtLayerPlugin.js

```diff
@@ -27,13 +29,45 @@ Oskari.clazz.defineES('Oskari.wfsmvt.WfsMvtLayerPlugin',
 
             sandbox.registerService(this.WFSLayerService);
             this.reqEventHandler = new ReqEventHandler(sandbox);
+
+            this._visualizationForm = Oskari.clazz.create(
+                'Oskari.userinterface.component.VisualizationForm'
+            );
         }
         _createPluginEventHandlers () {
             return Object.assign(super._createPluginEventHandlers(), this.reqEventHandler.createEventHandlers(this));
         }
         _createRequestHandlers () {
             return this.reqEventHandler.createRequestHandlers(this);
         }
+
+        /**
+         * @method getVisualizationForm
+         * @return {VisualizationForm} form
+         */
+        getVisualizationForm () {
+            return this._visualizationForm;
+        }
+
+        /**
+         * @method getLocalization
+         * Convenience method to call from Tile and Flyout
+         * Returns JSON presentation of bundles localization data for
+         * current language. If key-parameter is not given, returns
+         * the whole localization data.
+         *
+         * @param {String} key (optional) if given, returns the value for key
+         * @return {String/Object} returns single localization string or
+         *      JSON object for complete data depending on localization
+         *      structure and if parameter key is given
+         */
+        getLocalization (key) {
```

Comment:
We could refactor ShowOwnStyleRequestHandler.js so that we wouldn't need this method.